src/layout/layout.rs

```diff
@@ -27,7 +29,7 @@ type Rects = Rc<[Rect]>;
 // └   ┘└──────────────────┘└   ┘└──────────────────┘└   ┘└──────────────────┘└   ┘
 //
 // Number of spacers will always be one more than number of segments.
-type Cache = LruCache<(Rect, Layout), (Rects, Rects)>;
+type Cache = LruCache<(Rect, Layout), (Segments, Spacers)>;
```

Comment:
Does this need to be public?

Reply:
Let's leave it private for now till we figure out if there's a better API for returning these `Rects` to users.

Reply:
k